applications/HDF5Application/tests/test_hdf5_processes.py

```diff
@@ -0,0 +1,181 @@
+import os
+from KratosMultiphysics import *
+from KratosMultiphysics.HDF5Application import *
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+
+from single_mesh_temporal_output_process import Factory as OutputFactory
+from single_mesh_temporal_input_process import Factory as TimeInputFactory
+from single_mesh_initialization_process import Factory as InitializationFactory
+
+class TestHDF5Processes(KratosUnittest.TestCase):
+
+    def setUp(self):
+        self.model = Model()
+        self.buffer_size = 2
+
+    def tearDown(self):
+        pass
+
+    def testSingleMeshTemporalOutputInput(self):
+        """
+        Output ModelPart using SingleMeshTemporalOutputProcess
+        Use output file to set values in a new ModelPart with SingleMeshTemporalInputProcess
+        """
+        write_model_part = self._CreateNewModelPart("WriteModelPart")
+        self._InitializeModelPart(write_model_part)
+
+        read_model_part = self._CreateNewModelPart("ReadModelPart")
+
+        output_settings = Parameters(r"""{
+            "Parameters" : {
+                "model_part_name" : "WriteModelPart",
+                "nodal_solution_step_data_settings" : {
+                    "list_of_variables": ["VELOCITY", "DENSITY"]
+                },
+                "output_time_settings" : {
+                    "output_step_frequency": 1
+                }
+            }
+        }""")
+        output_process = OutputFactory(output_settings, self.model)
+
+        input_settings = Parameters(r"""{
+            "Parameters" : {
+                "model_part_name" : "ReadModelPart",
+                "nodal_solution_step_data_settings" : {
```

Comment:
Test would be more complete if we also included nodal and elemental data values.